Setup.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/env bash
+#*******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2018, 2024. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#*******************************************************************************
+# DBB Git Migration Modeler Configuration
+
+Prolog() {
+  echo "                                                                                                            "
+  echo " DBB Git Migration Modeler                                                                                  "
+  echo "                                                                                                            "
+  echo " Script:      Setup.sh                                                                                      "
+  echo "                                                                                                            "
+  echo " Description: This script is setting up a work folder including an configuration file that is leveraged     "
+  echo "              by the DBB Git Migration Modeler proces                                                       "
+  echo "                                                                                                            "
+  echo "              The user will be prompted for several configuration parameters that are saved                 "
+  echo "              into a configuration file within the work folder.                                             "
+  echo "              The configuration file is passed to the Migration Modeler as a required                       "
+  echo "              input parameter.                                                                              "
+  echo "                                                                                                            "
+  echo "              For more information please refer to:    https://github.com/IBM/dbb-git-migration-modeler     "
+  echo "                                                                                                            "
+}
+#
+Prolog
+
+# Default is the root of the Git Repo
+DBB_MODELER_HOME=$(cd "$(dirname "$0")" && pwd)
+
+# Configure DBB Migration Modeler Home
+echo "[INFO] Configure DBB Migration Modeler environment variables."
+
+echo "Please specify the DBB Migration Modeler home DBB_MODELER_HOME :"
```

Comment:
Could we have the prompts on one line ?
```suggestion
echo "Specify the DBB Migration Modeler Home [$DBB_MODELER_HOME] :"
```
This applies to all other prompts.